SecurityPkg/Tcg/Tcg2Pei/Tcg2Pei.c

```diff
@@ -259,7 +259,7 @@ EndofPeiSignalNotifyCallBack (
   }
 
   // MU_CHANGE - START
-  // Create a guid hob to save all excluded FVs for DXE - mschange start
+  // Create a guid hob to save all excluded FVs for DXE - MsChange start

```

Comment:
This has a MU_CHANGE tag above. We should try to be consistent with that moving forward.

Reply:
I made a bunch of these changes and missed this one.  I changed it to MU_CHANGE and removed the redundant label.